packages/gatsby-remark-prismjs/src/__tests__/__snapshots__/highlight-line-range.js.snap

```diff
@@ -1,87 +1,50 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`highlighting a line range highlight-line highlights line 1`] = `
-"<span class=\\"gatsby-highlight-code-line\\">  return \\"hello world\\"
-</span>"
-`;
+exports[`highlighting a line range highlight-line highlights line 1`] = `"<span class=\\"gatsby-highlight-code-line\\">  return \\"hello world\\"</span>"`;
```

Comment:
These diffs are a little messy, but it just removes the hard line breaks from the diffs. This will be handled via HTML start/end tags anyways, so this should be just fine.